Resources/doc/features/user-context.rst

```diff
@@ -36,7 +36,17 @@ These five steps resemble the Overview in the FOSHttpCache documentation.
 5. If the response to ``/foo`` should differ per user context, the application
    sets a ``Vary: X-User-Context-Hash`` header. The appropriate user context
    dependent representation of ``/foo`` will then be returned to the client.
+   
+.. note::
 
+    This concept basically executes a "preflight request" ("preflight" was term
```

Comment:
the spell checker does not like "preflight". should we write pre-flight or add this word to the spellchecker?